README.md

```diff
@@ -4,23 +4,99 @@
 
 ## 🧰 What does this project do?
 
-The Tuva Project's Readmissions data mart creates readmission flags on your patient population.  For a detailed overview of what the project does and how it works, check out our [Knowledge Base](https://thetuvaproject.com/docs/packages/readmissions).  For information on data models and to view the entire DAG check out our dbt [Docs](https://tuva-health.github.io/readmissions/#!/overview).
+The Tuva Project's Readmissions package calculates hospital readmission measures on your healthcare data.  
 
-## 🔌 Database Support
+For a detailed overview of the methodology used in the package check out our [Knowledge Base](https://thetuvaproject.com/docs/methodology/hospital-readmissions).  
+
+For information on data models and to view the entire DAG check out our dbt [Docs](https://tuva-health.github.io/readmissions/#!/overview).
+
+## 🔌 What databases are supported?
 
 This package has been tested on **Snowflake** and **Redshift**.
 
-## ✅ How to get started
+## 📚 What versions of dbt are supported?
+
+This package requires you to have dbt installed and a functional dbt project running on dbt version `1.2.x` or higher.
+
+## ✅ How do I use this dbt package?
+
+Below are the steps to run this individual dbt package.  To run all packages in The Tuva Project, please refer to this README.
+
+### Overview
+
+The Tuva Project is a collection of dbt packages that build healthcare concepts (measures, groupers, data quality tests) on top of your raw healthcare data. Each of these dbt packages expects you to have data in a certain format (specific tables with specific columns in them) and uses that as an input to then build healthcare concepts. To run any dbt package that is part of The Tuva Project, the basic idea is the same:
+
+1. You create the necessary input tables as models within your dbt project so that the Tuva package of interest can reference them using ref() functions.
+2. You import the Tuva package you are interested in and tell it where to find the relevant input tables as well as what database and schema to dump its output into.
+
+### **Step 1:**
+
+First you must create the necessary input tables as models within your dbt project so that the Tuva package of interest can reference them using ref() functions. To run this Readmissions package, you must create [these 4 tables](https://tuva-health.github.io/readmissions/#!/model/model.readmissions_input.condition) as models within your dbt project.
```

Comment:
Just wanted to mention this in case anyone else checks the link. This doesn't link to anything yet. I'm guessing it will once this PR is merged in and the Github page is updated with the latest version of the docs folder.